lib/internal/Magento/Framework/GraphQl/Exception/GraphQlAlreadyExistsException.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Framework\GraphQl\Exception;
+
+use GraphQL\Error\ClientAware;
+use Magento\Framework\Exception\AlreadyExistsException;
+use Magento\Framework\Phrase;
+
+/**
+ * Class GraphQlAlreadyExistsException
```

Comment:
More meaningful description is required for this and other newly introduced exceptions.